### DISCUSSION TITLE
i cant put the script into yimmenu

### DISCUSSION THREAD

**Discussion by blitzwall** [Deadlineem/Extras-Addon-for-YimMenu#82] [General]
i downloaded everything but i dont see extra addon script into my mod menu

**Comment by Deadlineem** [comment]
Did you go to settings, lua scripts and reload changed scripts checkbox then hit reload all?

If you still dont see the addon in your menu, press the open scripts folder button and open the 'scripts' folder, see if they are in there, if not you can manually download them or try running update.bat again, press 1, Y and hit enter

**Reply by blitzwall** [reply]
i see extra addon and extra data in the script folder

**Reply by Deadlineem** [reply]
> i see extra addon and extra data in the script folder

There should be 3 files in there, both data.lua and json.lua are required by extras-addon.lua

Extras-Addon.lua
Extras-Data.lua
Json.lua

**Reply by blitzwall** [reply]
it worked tysm i was wondering why i could not see it i been trying to figure it out for 3 day ty

**Reply by blitzwall** [reply]
i click on extra addon nothing pops up?

**Reply by Deadlineem** [reply]
> i click on extra addon nothing pops up?

If you add me on discord, i can further assist you in this, same username as on here.

**Comment by Deadlineem** [comment]
SOLVED:

Make sure Extras-Addon, Extras-data and json.lua are in the scripts folder and that they have their original names, the files cannot be found if they are named Extras-Addon (1).lua or something of the sort.
